.github/PULL_REQUEST_TEMPLATE.MD

```diff
@@ -0,0 +1,15 @@
+## Describe your changes
+
+
+## Describe testing performed
+*Please add any demo scripts to [CIL-Demons/misc/](https://github.com/TomographicImaging/CIL-Demos/tree/main/misc)*
+
+
+## Link relevant issues
+
+
+## Checklist before requesting a review
+- [ ] I have performed a self-review of my code
+- [ ] I have implemented unit tests that cover the new/modified functionality
+- [ ] CHANGELOG.md has been updated with any functionality change
```

Comment:
Maybe add some guidance on labelling?

Reply:
Maybe some extras such as:
- [ ] I have read and followed the developer guide: https://tomographicimaging.github.io/CIL/nightly/developer_guide.html
- [ ] I have added docstrings in line with the guidance in the developer guide

Reply:
doc strings! good plan.

Reply:
I've added a doc string check box, and a developer guide link